app/views/idv/hybrid_handoff/show.html.erb

```diff
@@ -0,0 +1,83 @@
+<% title t('titles.doc_auth.upload') %>
+
+<% content_for(:pre_flash_content) do %>
+  <%= render StepIndicatorComponent.new(
+        steps: Idv::Flows::DocAuthFlow::STEP_INDICATOR_STEPS,
+        current_step: :verify_id,
+        locale_scope: 'idv',
+        class: 'margin-x-neg-2 margin-top-neg-4 tablet:margin-x-neg-6 tablet:margin-top-neg-4',
+      ) %>
+<% end %>
+
+<%= render 'idv/doc_auth/error_messages', flow_session: flow_session %>
+
+<%= render PageHeadingComponent.new do %>
+  <%= t('doc_auth.headings.upload') %>
+<% end %>
+
+<p>
+  <%= t('doc_auth.info.upload') %>
+</p>
+
+<div class="grid-row grid-gap grid-gap-2">
+  <div class="grid-col-12 tablet:grid-col-auto">
+    <%= image_tag(
+          asset_url('idv/phone-icon.svg'),
+          alt: t('image_description.camera_mobile_phone'),
+          width: 88,
+          height: 88,
+        ) %>
+  </div>
+  <div class="grid-col-12 tablet:grid-col-fill">
+    <div class="usa-tag usa-tag--informative">
+      <%= t('doc_auth.info.tag') %>
+    </div>
+    <h2 class="margin-y-105">
+      <%= t('doc_auth.headings.upload_from_phone') %>
+    </h2>
+    <%= t('doc_auth.info.upload_from_phone') %>
+    <%= simple_form_for(
+          idv_phone_form,
+          as: :doc_auth,
+          url: url_for(type: :mobile, combined: true),
+          method: 'PUT',
+          html: { autocomplete: 'off' },
+        ) do |f| %>
+      <%= render PhoneInputComponent.new(
+            form: f,
+            required: true,
+            delivery_methods: [:sms],
+            class: 'margin-bottom-4',
+          ) %>
+      <%= f.submit t('forms.buttons.send_link') %>
+    <% end %>
+  </div>
+</div>
+
+<hr class="margin-y-4" />
+<div class="grid-row grid-gap grid-gap-2">
+  <div class="grid-col-12 tablet:grid-col-auto">
+    <%= image_tag(
+          asset_url('idv/laptop-icon.svg'),
+          alt: t('image_description.laptop'),
+          width: 88,
+          height: 88,
+        ) %>
+  </div>
+  <div class="grid-col-12 tablet:grid-col-fill">
+    <h2 class="margin-y-105">
+      <%= t('doc_auth.headings.upload_from_computer') %>
+    </h2>
+    <%= t('doc_auth.info.upload_from_computer') %>&nbsp;
+    <%= simple_form_for(
+          :doc_auth,
+          url: url_for(type: :desktop),
+          method: 'PUT',
```

Comment:
We discussed making this a link. Did we decide not to take that approach in this changeset?

Reply:
I decided against it for now because there were too many unknowns. For example, if we change how we are using the flow session to update/read the flow_path, we lose the ability to set it to `standard` explicitly in the form handler for manual upload (see `bypass_send_link_steps`). In the same vein, we would need to decide whether we really need an analytics call when users click that option, and, if so, whether that call would now happen on the frontend